src/Routes/DeviceDetail/DeviceDetail.js

```diff
@@ -126,12 +126,8 @@ const DeviceDetail = () => {
               {
                 title: 'Update',
                 isDisabled:
-                  imageData?.UpdateTransactions?.[
-                    imageData?.UpdateTransactions.length - 1
-                  ]?.Status === 'BUILDING' ||
-                  imageData?.UpdateTransactions?.[
-                    imageData?.UpdateTransactions.length - 1
-                  ]?.Status === 'CREATED' ||
+                  imageData?.UpdateTransactions?.[0]?.Status === 'BUILDING' ||
+                  imageData?.UpdateTransactions?.[0]?.Status === 'CREATED' ||
```

Comment:
now API returns the first update as the latest one

Reply:
Looks good to me, but I think explanatory comments like this should be in the code, commit description, PR description, and/or a Jira issue.